quaint/src/connector/postgres/conversion.rs

```diff
@@ -848,6 +848,16 @@ impl<'a> ToSql for Value<'a> {
             (Value::Array(vec), typ) if matches!(typ.kind(), Kind::Array(_)) => {
                 vec.as_ref().map(|vec| vec.to_sql(ty, out))
             }
+            (Value::EnumArray(variants, _), typ) if matches!(typ.kind(), Kind::Array(_)) => variants
+                .as_ref()
+                .map(|vec| vec.iter().map(|val| val.as_ref()).collect::<Vec<_>>().to_sql(ty, out)),
+            (Value::EnumArray(variants, _), typ) => {
+                let kind = ErrorKind::conversion(format!(
+                    "Couldn't serialize value `{variants:?}` into a `{typ}`. Value is a list but `{typ}` is not."
+                ));
```

Comment:
I'm curious why we don't go with `r#type`?
That's what we've been going with in PSL at least and I see it's usage in quaint and QE too

Reply:
Because I dislike having to type r# every single time (even though the autocompletion does the job) and find it ugly 🙈. A middle ground I can live with is `type_`, but I usually find `typ` to be more straightforward.